authors/apps/articles/models.py

```diff
@@ -71,3 +72,23 @@ def __str__(self):
 
 # register the pre_save signal
 pre_save.connect(Article.pre_save, Article, dispatch_uid="authors.apps.articles.models.Article")
+class ArticleImage(BaseModel):
```

Comment:
Did you work on this??

Reply:
This is from Moses branch....I rebased from his branch